src/rgw/rgw_lc_s3.cc

```diff
@@ -139,7 +139,16 @@ void LCFilter_S3::decode_xml(XMLObj *obj)
     single_cond = true;
   }
 
-  RGWXMLDecoder::decode_xml("Prefix", prefix, o);
+  {
+    auto prefix_iter = o->find("Prefix");
+    auto prefix_val = prefix_iter.get_next();
+    if(prefix_val) {
+      decode_xml_obj(prefix, prefix_val);
+      if(prefix_iter.get_next()){
+        throw RGWXMLDecoder::err("Bad filter: the number of prefix is more than one");
```

Comment:
For proper English, I would say: "the number of prefixes is more than one".

For more natural English, I would say: "there is more than one prefix".
Or even better: "there should not be more than one prefix".